### DISCUSSION TITLE
Newbie new install with no libraries

### DISCUSSION THREAD

**Discussion by logan70389** [Sloeber/arduino-eclipse-plugin#1646] [General]
I just did an install and all my includes have the 'unresolved inclusion' error.  It seems my libraries are in:
D:\Sloeber\arduinoPlugin\libraries. I tried  adding this to libraries but got message that this is not allowed.
I have selected a 
board -ATmega2560
programmer protocol ArduinoIISP
Platform Folder D:\Sloeber\arduinoPlugin\packages\arduino\hardware\avr\1.8.6

Project Include Directories D:\Sloeber\arduinoPlugin\libraries

I've run out of ideas on what to check. Any thoughts on how to resolve this?
Thanks
Tom

**Comment by jantje** [comment]
Hi
A good starting point is the document provided here
https://github.com/Sloeber/arduino-eclipse-plugin/discussions/1342
Normally Sloeber should add the libraries to the project automagically (if they are installed)
If they are not installed you can find the library manger under the menu item sloeber->preferences
The first thing that comes to mind is: are you in the sloeber perspective?
![image](https://github.com/Sloeber/arduino-eclipse-plugin/assets/1175841/de7b26f7-3dfd-48fd-9286-0a4335b7236b)

**Comment by logan70389** [comment]
I double checked and I am in the Sloeber perspective as far as I know. I did a change perspective to Sloeber, saved and then did a 'reattach' libraries with no success. Here's an image of what I have setup
![image](https://github.com/Sloeber/arduino-eclipse-plugin/assets/6496152/91d50628-4f9a-4fa1-a15f-18c3c16e0c68)

The c:/Users/tlogaan/Documents/Arduino/library directory has nothing. My Sloeber libraries are in D:\Sloeber\arduinoPlugin\libraries where the install is located. If I try to add that directory to that view - I get an error telling me I'm not allowed to do that.
Thanks for help and quick response - I've been out last few hours
Tom

**Comment by logan70389** [comment]
I got it working by deleting it and doing another install.  I'm off and running now!
thanks
Tom

**Comment by jantje** [comment]
There were some errors in your config
![image](https://github.com/Sloeber/arduino-eclipse-plugin/assets/1175841/9f3b69fa-35e0-4146-af8e-8f3b410e5e3c)
Great it is working now.
